index.html

```diff
@@ -213,45 +208,52 @@ <h4>
             targets are available, or the identity of the chosen target.
           </p>
           <div class="note">
-            <a for="Navigator"><code>share</code></a> always shows some form of
-            UI, to give the user a choice of application and get their approval
-            to invoke and send data to a potentially native application (which
-            carries a security risk). For this reason, user agents are
-            prohibited from showing any kind of "always use this target in the
-            future" option, or bypassing the UI if there is only a single share
-            target.
+            <a>share()</a> always shows some form of UI, to give the user a
+            choice of application and get their approval to invoke and send
+            data to a potentially native application (which carries a security
+            risk). For this reason, user agents are prohibited from showing any
+            kind of "always use this target in the future" option, or bypassing
+            the UI if there is only a single share target.
           </div>
         </section>
       </section>
-      <section data-dfn-for="ShareData">
+      <section data-dfn-for="ShareData" data-link-for="ShareData">
         <h3>
           <code>ShareData</code> dictionary
         </h3>
         <pre class="idl">
           dictionary ShareData {
-            USVString title;
             USVString text;
+            USVString title;
             USVString url;
           };
         </pre>
         <p>
           The <dfn>ShareData</dfn> dictionary consists of several optional
-          fields:
+          members:
```

Comment:
In the spec, you use "fields" throughout... but in IDL, I think these are called members.

Reply:
Will file a separate bug for this.

Reply:
Ack.